db/schema.rb

```diff
@@ -10,7 +10,17 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180323100630) do
+ActiveRecord::Schema.define(version: 20180323103213) do
+
+  create_table "posts", force: :cascade do |t|
+    t.text "content"
+    t.integer "upvotes"
+    t.integer "downvotes"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer "user_id"
+    t.index ["user_id"], name: "index_posts_on_user_id"
```

Comment:
Do you know why index was generated here?

Reply:
@talihomz @SavaVuckovic this is an important question :) Please answer 🙏

Reply:
I'm guessing its job is to speed up lookup for posts belonging to certain users? 😅